install/upgrade_to_2.13.10.php

```diff
@@ -0,0 +1,79 @@
+<?php
```

Comment:
It seems simpler and less risky not to resize at all at the update, neither in this version nor in 2.13.7.